_layouts/partners/roadmap.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: base
+---
+<header>
+  {% include skip_nav.html %}
+  {% include banner.html %}
+  {% include partners/header.html %}
+</header>
+<main id="main-content">
+  <section>
+    <div class="container partners-container">
+      <div class="grid-row">
+        <h1 class="tablet:grid-col-6 partners-title">{{ page.title }}</h1>
+        <div class="tablet:grid-col-12 partners-border"></div>
+        <div class="roadmap-body grid-row">
+          <div class="text tablet:grid-col-7 grid-col-12">
+            <div>{{ page.subtitle | markdownify}}</div>
+            <div class="partners-subtitle">{{ page.subsection | markdownify }}</div>
+          </div>
+          <div class="image tablet:grid-col-5 grid-col-12">
+            <a aria-label="Program Roadmap" href="{{ page.roadmap-url}}"><img class="section-title-right" alt="Program Roadmap" src="{{ site.baseurl }}/assets/img/partners/roadmap.png"/></a>
```

Comment:
I'm getting 404 for the link when I click on it on the federalist build

Reply:
Probably because the preview branch has subpaths, so root-relative paths don't work. The link helper should be used to resolve to the full URL.

Untested, but should work:

```suggestion
            <a aria-label="Program Roadmap" href="{% link page.roadmap-url %}"><img class="section-title-right" alt="Program Roadmap" src="{{ site.baseurl }}/assets/img/partners/roadmap.png"/></a>
```